homeassistant/components/aemet/const.py

```diff
@@ -202,43 +202,43 @@
 
 FORECAST_SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
```

Comment:
Side note: A nice follow up would be to move the sensor descriptions and monitored conditions to the sensor platform.